content/applications/sales/sales/send_quotations/optional_products.rst

```diff
@@ -1,65 +1,67 @@
 
-   .. image:: optional_products/optional_products_3.png
+   .. image:: optional_products/optional-products-3.png
       :align: center
-      :class: img-thumbnail
-      :alt: Preview your quotations on Odoo Sales
+      :alt: Preview your quotations on Odoo Sales.
+
+In practice, customers are able to add different optional products to an order by clicking the
+:guilabel:`Shopping Cart icon`, located to the right of the optional product line.
 
-In practice, your customers will be able to add different optional products to their order by
-using associated carts, with a user-friendly layout. More than that, if a customer selects all the
-optional products suggested, these additional items will automatically fill in the quotation
-managed by the salesman.
+Additionally, if a customer selects all the optional products suggested, these additional items are
+automatically filled in the quotation managed by the salesman.
 
-.. image:: optional_products/optional_products_4.png
+.. image:: optional_products/optional-products-4.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to select optional products on Odoo Sales
+   :alt: How to select optional products on Odoo Sales.
 
-Like this, salespeople will see each movement made by the customer and tracking the order will be
-all the better.
+When the customer adds an optional product(s) to an order, the salesperson sees each movement made
+by the customer, providing them with accurate tracking of the entire order in the backend of the
+*Sales* application (on the specific quotation form).
 
-.. image:: optional_products/optional_products_5.png
+.. image:: optional_products/optional-products-5.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to follow each movement made by your customers on Odoo Sales
+   :alt: How to follow each movement made by your customers on Odoo Sales.
 
-Add optional products to your quotation templates
-=================================================
+Optional products on quotation templates
+========================================
 
-Be sure to check out our documentation about :doc:`quote_template` to understand how you can
-enable, create, design and manage your own quotation templates before reading this part.
+Be sure to review the documentation about
+:doc:`/applications/sales/sales/send_quotations/quote_template` to better understand how quotation
+templates can be enabled, created, designed, and managed before reading the following.
 
-For quotation templates, you also have an **Optional Products** tab where you can add related
-products or services.
+For quotation templates, just like a typical quotation form, there is also an :guilabel:`Optional
+Products` tab, wherein related products or services can be added to a quotation template. Those
+products added under the :guilabel:`Optional Products` tab are present in the quotation, by
+default, whenever that particular quotation template is used.
```

Comment:
early 100th char line break

Reply:
I see. I only made this break because I thought that since the comma was right after the word "default" (in other words no spaces separating the two) that it needed to be knocked down to the next line. No worries, though. I slightly restructured the sentence to make everything work. Thanks 👍